docs/end-user.rst

```diff
@@ -192,6 +192,64 @@ steps:
 
 There are also two other navigation options to find content.
 
+Move content between Tiles
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To manage a big site you will eventually need to change an important new from
+one place to another of your Cover.
+
+If you move the content data from one tile to another, it will behave just like
+if you deleted the content from the first tile and added the content data into
+the second tile (this feature works for every tile created after version 1.0a13).
```

Comment:
I didn't get your point.. where in this text we are talking about recreate the whole front page?!